### PR TITLE
MODSOURMAN-553 Update GET /change-manager/parsedRecords to have externalId param

### PR DESCRIPTION
**Pull request by Shans-Kaluhin**
## Purpose
Update GET /change-manager/parsedRecords  to have externalId param instead instanceId. It will give quick-marc an ability to fetch records by its externalId from SRS.

## Approach
instanceId parameter is replaced with externalId 

## Learning
[MODSOURMAN-553](https://issues.folio.org/browse/MODSOURMAN-553)
[API Documentation](https://s3.amazonaws.com/foliodocs/api/mod-source-record-manager/change-manager.html#change_manager_parsedrecords_get)

### CODE REVIEW

File: descriptors/ModuleDescriptor-template.json [hunk 1]
```diff
@@ -446,7 +446,7 @@
     },
     {
       "permissionName": "change-manager.parsedrecords.get",
-      "displayName": "Change Manager - get parsed records by instanceId",
+      "displayName": "Change Manager - get parsed records by externalId",
```
Comment (psmagin) [inline descriptors/ModuleDescriptor-template.json:449]: Update major interface version of interface with this endpoint

Comment (psmagin) [reply descriptors/ModuleDescriptor-template.json:449]: @natali-zaitseva, what we should do in case of a major version change?

Comment (psmagin) [reply descriptors/ModuleDescriptor-template.json:449]: Thank you!
As I can see only mod-quick-marc and ui-agreements are using this interface. 
image

Comment (natali-zaitseva) [reply descriptors/ModuleDescriptor-template.json:449]: and we should it before R3 "Major interface provided to be updated", which seems to be before 17 Sep.

Comment (psmagin) [reply descriptors/ModuleDescriptor-template.json:449]: I notified ERM team directly about the change. But we still could notify #development channel.

**Review by psmagin — APPROVED** [review]

**Review by natali-zaitseva — APPROVED** [review]

**Pull request merged**